api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -623,10 +607,105 @@ public void testCloneWorkspaceWithCohorts() throws Exception {
     CohortReview cr1 = cohortReviewController.createCohortReview(
         workspace.getNamespace(), workspace.getId(), c1.getId(),
         cdrVersion.getCdrVersionId(), reviewReq).getBody();
+    CohortAnnotationDefinition cad1EnumRequest = new CohortAnnotationDefinition()
```

Comment:
opt_nit: Personal preference, but I might inline these request objects into the API call invocation throughout as they aren't used otherwise.